staubli_val3_driver/val3/ros_server/pushMotion.pgx

```diff
@@ -65,36 +65,36 @@
   endSwitch
 
   // see github.com/ros-industrial/industrial_core/simple_message/include/simple_message/joint_traj_pt.h
-  if (l_mTrajPt.nSequence==-4)
+  if(l_mTrajPt.nSequence==-4)
     // set flag to stop immediately
     bStopNow=true
   else
     // check whether velocity value should be overwritten with value set on teach pendant
-    if (bOverwriteVel == true)
+    if(bOverwriteVel == true)
       // overwrite velocity (and acceleration - ?)
-      l_mTrajPt.mDesc.vel = getMonitorSpeed()
+      //l_mTrajPt.mDesc.vel = getMonitorSpeed()
       // explicitly setting accel and decel to VAL3 default values
-      l_mTrajPt.mDesc.accel = 9999
-      l_mTrajPt.mDesc.decel = 9999
+      l_mTrajPt.mDesc.accel = mNomSpeed.accel
+      l_mTrajPt.mDesc.decel = mNomSpeed.decel
```

Comment:
This will automatically use whatever is default for the controller this is being loaded on, correct?

Reply:
mNomSpeed will be a part of the application, so it would need to be changed by the user after loading the program. It was the only way that I could think of to have a "velocity overwrite" since using the +/- buttons on the pendant don't have the desired effect here. Maybe some documentation should be added to the readme to make that clear? IMO, the velocity overwrite should be removed entirely, but I did not want to do that without additional input.

Reply:
We've discussed this in #1. I would vote for removing it.

We can always bring it back later.

Reply:
We've discussed this in #1. I would vote for removing it.

We can always bring it back later.

Reply:
Tracked in #1.